packages/ai-mcp/tests/cli-introspect.test.ts

```
@@ -0,0 +1,14 @@
+import { describe, expect, it } from 'vitest'
+import { makeFullServer } from './helpers/in-memory-server'
+import { introspectFromTransport } from '../src/cli/introspect'
```

ESLint

[error] `../src/cli/introspect` import should occur before import of `./helpers/in-memory-server`

(import/order)